app/models/organization.rb

```diff
@@ -223,6 +223,13 @@ def seed_items(item_collection)
     reload
   end
 
+  def seed_random_item_with_name(name)
```

Comment:
This shouldn't go on Organization if it's only used to create seed data. It should be a method in `seeds.rb`.

Reply:
Will do. Am I correct in assuming I should then discard the associated unit test, since the other functions in `seed.rb` lack them?

Reply:
Correct.